docs/guide/use-your-own-domain.md

```diff
@@ -395,3 +395,61 @@ urlpatterns = [
     path("@${MASTODON_USER}", blog.username_redirect),
 ]
 ```
+
+### Remix (JavaScript)
+
+!!! info "Original implementation by [Tom Sherman](https://tom-sherman.com/blog/mastodon-domain-alias-on-remix){target=_blank}, please see his blog post for more information"
+
+!!! note "This configuration will only redirect the "me" username on `@${ALIAS_DOMAIN}` to your Mastodon account `${MASTODON_USER}@${MASTODON_DOMAIN}`"
+
+Add a new route at `app/routes/[.]well-known/webfinger.js` with content as follows:
+
+```js
+export function loader({ request }) {
+  const url = new URL(request.url);
+  const resourceQuery = url.searchParams.get("resource");
+
+  if (!resourceQuery) {
+    return new Response("Missing resource query parameter", {
+      status: 400,
+    });
+  }
+
+  if (resourceQuery !== "acct:me@tom-sherman.com") {
```

Comment:
Maybe add a comment that you can remove this `if` clause if you want `*@${ALIAS_DOMAIN}` redirection